recipes/atool2/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "atool2" %}
+{% set version = "0.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/solsticedhiver/atool2/archive/{{ version }}.tar.gz
+  sha256: 87fc55cc21e2ac74c356797fc4e1d73b37dd37a6b91811cea778c68787079239
+
+build:
+  # Missing automake
+  skip: true  # [win]
+  number: 0
+  # noarch: generic # Re-enable after fixing Windows build
```

Comment:
Note that noarch packages are built on Linux and installed on macOS and Windows. What is happening with the Windows build here? Anything that is arch specific won't allow this to become a noarch package later.

Reply:
Could it be the case that `noarch: generic` packages are not built on Linux? This is the failed pipeline https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=991382&view=logs&j=240f1fee-52bc-5498-a14a-8361bde76ba0&t=7c0f8eae-6d6f-51bf-636f-73a1a7fb1bc4

Reply:
It is missing automake:

```
Encountered problems while solving:
  - nothing provides requested automake
```

> Could it be the case that noarch: generic packages are not built on Linux?

They are built only on Linux in the feedstock. If you are sure this can be noarch generic, we can ignore that Windows failure and make it noarch here. We just need to be sure there are no compiled stuff there and that the files will be placed in sane directories for all OSes.

Reply:
The original `atool` seems to be a perl script dropped in a bin directory, right? Did you ever try to use that on Windows? My guess is that it won't work b/c it should be in the `%PREFIX%\Library\bin`. On Linux and mac that is `$PREFIX/bin`.

TL;DR I'm not sure how to best make this noarch generic. You can try OS-noarch and build it once for Linux+mac and they another build for Windows. However, all that won't matter here b/c your Windows build is not working. Can you propagate this comment to an issue in the feedstock. We can merge this as-is for now.